src/base/routes/main.js

```diff
@@ -248,46 +248,46 @@ export default [
         },
       },
       {
-        name: 'stores',
-        path: 'store',
+        name: 'places',
+        path: 'place',
```

Comment:
I'd like to keep public URLs stable, so maybe add a redirect/alias from `store`?